tools/testing/selftests/kvm/priv_memfd_test.c

```diff
@@ -17,9 +17,19 @@
 #include <kvm_util.h>
 #include <processor.h>
 
+#define BITS_IN_BYTE 8
```

Comment:
@adiviness : Let's try to find the replacement for this macro available from standard c libraries or refer to other instances in the selftests.